server/routes/support/handlers/offenderDetail.ts

```diff
@@ -18,7 +23,43 @@ export default class OffenderDetailRoutes {
       ? await this.communityService.getStaffDetailByStaffCode(probationPractitioner?.staff.code)
       : undefined
     const hdcStatus = _.head(await this.prisonerService.getHdcStatuses([prisonerDetail], user))
+    const licenceSummary = await this.licenceServer.getLatestLicenceByNomisIdsAndStatus([nomsId], [], user)
 
+    let licenceFromSummary
+    let licencecrn = 'Not found'
```

Comment:
I don't think we need CRN at all here, as it's a Delius identifier rather than a licence date.

Reply:
Done.